package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "panda-sky",
-  "version": "1.0.0-beta-16",
+  "version": "1.0.0-beta-18",
   "description": "Quicky publish severless applications in the cloud",
   "main": "lib/sky-helpers/index.js",
   "bin": {
```

Comment:
This should not be `bin` but rather `files`. See:

https://docs.npmjs.com/files/package.json#files